std_msgs/Header header
uint32 id
string class_name
float32 confidence
float32 dx
sensor_msgs/RegionOfInterest bbox_2d
geometry_msgs/Point point_3d